ui/admin/app/templates/scopes/scope/groups/index.hbs

```diff
@@ -31,42 +31,42 @@
 
     {{#if @model}}
 
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model as |group|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if (can 'read model' group)}}
-                  <LinkTo
-                    @route='scopes.scope.groups.group'
-                    @model={{group.id}}
-                  >
-                    {{group.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{group.displayName}}
-                {{/if}}
-                <p>{{group.description}}</p>
-              </row.headerCell>
-              <row.cell>
-                <Copyable
-                  @text={{group.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
-                >
-                  <code>{{group.id}}</code>
-                </Copyable>
-              </row.cell>
-            </body.row>
-          {{/each}}
-        </table.body>
-      </Rose::Table>
+      <Hds::Table
+        class='groups-list-table'
```

Comment:
do we need a classname for this?

Reply:
I don't know I just added it since the reference I used had that.

Reply:
looks like it's not being used anywhere. can we remove this class and also the `users-list-table` from the users list?